weblate/checks/consistency.py

```diff
@@ -111,6 +116,13 @@ def check_target_unit(self, sources, targets, unit):
 
         for action in changes.values_list("action", flat=True):
             if action in Change.ACTIONS_CONTENT:
+                self.description = (
+                    _(
+                        'This string has been translated in the past. \
+                                Last Translation was "%s"'
+                    )
+                    % changes.latest("timestamp").target
```

Comment:
This hits again the database making the check twice slower than necessary. Better approach would be to include target in values_list above and use the value here directly without additional SQL query.

Also changing description class attribute doesn't sound right. Better approach might be to return the target value here and generate the verbose description in the get_description method.